packages/headless/src/controllers/folded-result-list/headless-folded-result-list.ts

```diff
@@ -148,111 +42,18 @@ export interface FoldedResultListState extends SearchStatusState {
  */
 export function buildFoldedResultList(
   engine: SearchEngine,
-  props: FoldedResultListProps = {}
+  props: FoldedResultListProps
```

Comment:
why do we need to remove this default value?

Reply:
I think this is a breaking change, we shall not touch the signature of the `buildFoldedResultList`.